src/api/replay.js

```diff
@@ -225,6 +239,7 @@ const InsertReplay = async (data, userid) => {
     insertData.DrivenByText = '';
   }
   const replay = await Replay.create(insertData);
+  await replay.setTags(insertData.Tags.map(tag => tag.TagIndex));
```

Comment:
Here should probably add `DNF` tag automatically for replay if it isn't finished.

Reply:
Should also check that `Hidden` tags are not added.